internal/dataplane/kongstate/route_test.go

```diff
@@ -1055,3 +1055,131 @@ func TestOverrideHosts(t *testing.T) {
 		})
 	}
 }
+
+func TestOverrideHeaders(t *testing.T) {
+	type args struct {
+		route Route
+		anns  map[string]string
+	}
+	tests := []struct {
+		name string
+		args args
+		want Route
+	}{
+		{name: "basic empty route"},
```

Comment:
nit: what would you say of being consistent with using new lines here?

```suggestion
		{
			name: "basic empty route",
		},
```